Raspi/Test/mock_wireless_module.py

```diff
@@ -0,0 +1,170 @@
+# MQTT wireless sensor test script that sends fake randomly generated data
+# To do a single test --> python3 mqtt_wireless_test.py -t 0
```

Comment:
No need to write this! It is assumed to the user that they will need to type in `python3 mock_wireless_module --help` that they will be shown how to use this script!